tests/integration/cairo1_hint_locals/prank/prank_test.cairo

```diff
@@ -0,0 +1,88 @@
+use array::ArrayTrait;
+use result::ResultTrait;
+
+#[test]
+fn test_prank() {
+    let deployed_contract_address = deploy_contract('pranked', ArrayTrait::new()).unwrap();
+
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 0, 'check call result');
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 2, 'check call result');
+
+    start_prank(123, deployed_contract_address).unwrap();
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 3, 'check call result');
+
+    stop_prank(deployed_contract_address).unwrap();
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 2, 'check call result');
+}
+
+#[test]
+fn test_stop_prank_on_non_existent() {
+    stop_prank(1234).unwrap();
+}
+
+#[test]
+fn test_stop_prank_on_not_pranked() {
+    let deployed_contract_address = deploy_contract('pranked', ArrayTrait::new()).unwrap();
+
+    stop_prank(1234).unwrap();
+
+    invoke(deployed_contract_address, 'maybe_set_three', ArrayTrait::new()).unwrap();
+    let return_data = call(deployed_contract_address, 'get_a', ArrayTrait::new()).unwrap();
+    assert(*return_data.at(0_u32) == 2, 'check call result');
+}
+
+#[test]
+fn test_stop_prank_multiple_times() {
+    let deployed_contract_address = deploy_contract('pranked', ArrayTrait::new()).unwrap();
+    start_prank(123, deployed_contract_address).unwrap();
+
+    stop_prank(deployed_contract_address).unwrap();
+    stop_prank(deployed_contract_address).unwrap();
```

Comment:
shouldn't that fail?

Reply:
Previously `start_prank` returned a callable that could cancel a prank. I'm assuming it was impossible for it to fail then. 

Having the ability to call `stop_prank` multiple times/on non-existing is just the behavior I chose.

Reply:
If others agree about this, i'm fine with it as well, don't have a strong opinion on this

Reply:
In my opinion 
- pranking non existent contract makes sense, you might want to prank constructor on deploy
- stop_prank - when it cleans up everything i would also let it run on unpranked contracts without error. I interpret the semantics as - clean everything up if there is something to clean up

Reply:
Good point @Arcticae, but I don't see a reason why to not add it here